lib/applications/types.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * applications object type used in csv generatiion
+ */
+export type GenerateApplicantsReportApplications = {
```

Comment:
lol does anyone have better name suggestions

Reply:
See comment above in `lib/applications/resolvers.ts`, I don't think we'll need this anymore

Reply:
Removed!